fe/topology.py

```diff
@@ -253,11 +303,125 @@ def parameterize_periodic_torsion(self, proper_params, improper_params):
         improper_params, improper_potential = self.parameterize_improper_torsion(improper_params)
         combined_params = jnp.concatenate([proper_params, improper_params])
         combined_idxs = np.concatenate([proper_potential.get_idxs(), improper_potential.get_idxs()])
-        combined_potential = potentials.PeriodicTorsion(combined_idxs)
+
+        # dual topology needs this fix too... probably
```

Comment:
This is an oversight on my part, definitely something I forgot to translate over. Will fix in this patch.

Reply:
PS in case you were wondering why the tests I showed earlier still worked fine, in my rabfe_jank branch, I used direct parameter replacement (i.e. setting k_torsion == 0). But while refactoring this code, I wanted to be consistent with the BaseTopology and instead modified this to use lambda_offset indices etc, and I only patched the BaseTopology parameterize_periodic_torsion, not the DualTopology variant.

Thanks for catching this!